man/systemd.exec.xml

```diff
@@ -1311,6 +1311,19 @@
         <citerefentry><refentrytitle>tmpfiles.d</refentrytitle><manvolnum>5</manvolnum></citerefentry>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>DenyWriteExecuteMemory=</varname></term>
+
+        <listitem><para>Takes a boolean argument. If set, a seccomp filter is added to reject
+        <citerefentry><refentrytitle>mmap</refentrytitle><manvolnum>2</manvolnum></citerefentry>
+        system calls with both <constant>PROT_EXEC</constant> and <constant>PROT_WRITE</constant> set
+        and <citerefentry><refentrytitle>mprotect</refentrytitle><manvolnum>2</manvolnum></citerefentry>
+        system calls with <constant>PROT_EXEC</constant> set. This option may be incompatible with dynamic
+        code generators, programs using trampolines and some Xorg drivers.
```

Comment:
This should really start with an explanation in human readable language, i.e. something like:

> Takes a boolean argument. If set, attempts to create memory mappings that are writable and executable at the same time, or to change existing memory mappings to become executable are prohibited. Specifically, a system call filter is added that rejects…

This should be followed by the low-level explanation you already have in place. But I am pretty sure it shouldn't mention "seccomp", as that's really an implementation detail...

Also, the option *is* incompatible with dynamic code generators, it's not just a "may be". And the trampoline thing might be amusing a bit I think without any context.

Maybe: "Note that this option is incompatible with programs that generate program code dynamically at runtime, such as JIT execution engines, or programs compiled making use of the code "trampoline" feature of various C compilers. 

Finally, and most importantly: this should really say what the benefit of this option is: "this option improves service security, as it makes software exploits involving changing running code dynamically harder." or something like that...